### PR TITLE
Added ARC Informatique and Opto 22

### PR DESCRIPTION
**Pull request by inl-ics**
#  #

## ✅ Pre-approval checklist ##

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - _eschew scope creep!_
- [ ] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

**Comment by inl-ics** [comment]
what is lint?
\

**Comment by inl-ics** [comment]
looks like a indentation issue, is is with the affected versions? I wasn't sure if the - should be inline or should the text be inline and the - be further out.

File: data/cisagov_O.yml [hunk 1]
```diff
@@ -586,6 +586,130 @@ software:
...
+      cve-2021-4104:
+        investigated: false
+        affected_versions: []
+        fixed_versions: []
+        unaffected_versions: []
+      cve-2021-44228:
+        investigated: true
+        affected_versions:
+          - < 3.3.2
+        fixed_versions:
```
Comment (Lcerkov) [inline data/cisagov_O.yml]:
```suggestion
        fixed_versions:
```

File: data/cisagov_O.yml [hunk 2]
```diff
@@ -586,6 +586,130 @@ software:
...
+      cve-2021-4104:
+        investigated: false
+        affected_versions: []
+        fixed_versions: []
+        unaffected_versions: []
+      cve-2021-44228:
+        investigated: true
+        affected_versions:
+          - < 4.3g
+        fixed_versions:
```
Comment (Lcerkov) [inline data/cisagov_O.yml]:
```suggestion
        fixed_versions:
```

File: data/cisagov_O.yml [hunk 3]
```diff
@@ -586,6 +586,130 @@ software:
...
```
Comment (Lcerkov) [inline data/cisagov_O.yml]:
```suggestion
        fixed_versions:
```

File: data/cisagov_O.yml [hunk 4]
```diff
@@ -586,6 +586,130 @@ software:
...
```
Comment (Lcerkov) [inline data/cisagov_O.yml]:
```suggestion
        fixed_versions:
```

**Comment by Lcerkov** [comment]
@inl-ics Looks like a trailing white space error. I think I found and corrected the white spaces if you want to accept those changes.

**Comment by Lcerkov** [comment]
@inl-ics I went ahead and committed the changes (deleting trailing white spaces) and am merging this PR.

**Pull request merged**